data_ingestion/common/gcf_household_income.py

```diff
@@ -0,0 +1,36 @@
+from .di_url_file_to_gcs import url_file_to_gcs
+
+
+def get_columns():
```

Comment:
As far as I can tell, the api requires the user to explicitly list each column and doesn't have a "select all" feature. I excluded some data for child poverty by age group in this iteration because I wasn't sure it was relevant to the prototype. I think we can add it later if we decide it will be useful.

Reply:
Ah ok sounds good.